share/picongpu/examples/LaserWakefield/lib/python/picongpu/params.py

```diff
@@ -32,7 +32,7 @@
             default=800.0, range=(400.0, 1400.0)),
 
         Parameter(
-            name="Pulse_Length_SI", ptype="compile", unit="fs",
+            name="Pulse_Duration_SI", ptype="compile", unit="fs",
```

Comment:
@BrianMarre is this consistent with PICMI?

Reply:
ping @BrianMarre

Reply:
this file has nothing to do with PICMI/PyPIConGPU, all PyPIConGPU stuff in share is under 'share/picongpu/pypicongpu'

Reply:
The PyPIConGPU file you need to update is this [one](https://github.com/ComputationalRadiationPhysics/picongpu/blob/15e29cee4f688d59c08ed7c713ed96be2b49d00f/share/picongpu/pypicongpu/template/include/picongpu/param/incidentField.param.mustache)

Reply:
OK than this is from "Eupraxia"

Reply:
I already changed this file! See (https://github.com/ComputationalRadiationPhysics/picongpu/blob/25c9a35da92ab0f92fce5159c656e4b410c937bf/share/picongpu/pypicongpu/template/include/picongpu/param/incidentField.param.mustache)